backend/app/modules/uq/core.py

```diff
@@ -0,0 +1,35 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Scores:
+    identity_poap: int = 10
+    epoch0_poap: int = 10
+    num_of_donations: int = 10
+    threshold: int = 20
+    min_return: float = 0.2
+    max_return: float = 1.0
+
+
+def calculate_uq(
+    has_epoch_zero_poap: bool,
+    has_identity_poap: bool,
+    num_of_donations: int,
+    gp_score: float,
+    scores: Scores,
+) -> float:
+    uq_score = 0
+
+    if has_identity_poap:
```

Comment:
it is probably misleading as well, if we won't issue a poap

Reply:
changed